clean/ca/los_angeles_sheriff.py

```diff
@@ -0,0 +1,248 @@
+        r = requests.post(indexjsonurl, headers=indexrequestheaders, data=indexpayload)
+        with open(self.cache_dir / (self.siteslug + "/index.json"), "wb") as outfile:
```

Comment:
Was there an issue with [`self.cache.write_json`](https://github.com/biglocalnews/clean-scraper/blob/ca-51/clean/cache.py#L161)? If possible, let's replace instances of `open` with the relevant methods from the Cache class @zstumgoren wrote

Reply:
The existing JSON cache read and write methods are hardcoded to work only lists of dicts, from what I can tell, and this is not a list of dicts.

Reply:
I'm not following 😅 Could you link me to where those methods are hardcoded, please? I think I can make suggested edits to adapt the existing cache read and write methods

Reply:
I may be misunderstanding, but:
def write_json(self, name: Union[Path, str], files_meta: list[dict]) -> Path:
...Args
...            content (list[dict]): List of dicts containing file metadata for downloadable assets

 def read_json(self, name: Path) -> list[dict]:
...
        Returns:
            list of dicts

Reply:
Oh, the type signatures. Hm, does your scraper and San Diego's still work as expected if those signatures are removed or updated to accommodate different JSON types?

Reply:
Resolved with https://github.com/biglocalnews/clean-scraper/pull/68